### ISSUE TITLE
Accessibility in submit button

### ISSUE THREAD

**Issue by diogojorgebasso** [LucasBassetti/react-simple-chatbot#313]
**Describe the bug**
I was running Accessibility Insights for Web, from Microsoft, and accused that the submit button does not have inner text that is visible to screen readers or aria-label attribute.
Thus, I made a very simple PR for fixing the issue, adding aria-label
**To Reproduce**

1. Make an Accessibility test
2. Found the error

**Expected behavior**
Add this property.

**Screenshots**
![image](https://user-images.githubusercontent.com/65865529/127746620-5aac7e67-17ee-415b-8632-5b2b34692587.png)

**Desktop (please complete the following information):**
 - All Browsers
 - Version 0.6.1
